### ISSUE TITLE
display.hex(0xbeef)

### ISSUE THREAD

**Issue by richpaul6806** [bablokb/circuitpython-tm1637#5]
Does this serve any purpose?

**Comment by bablokb** [comment]
Yes, it should display the word "beef" on the display. It is actually a test that display.hex() is working.

**Comment by richpaul6806** [comment]
Thanks. I figured it was something like that. It made more sense once I finally got it working and it only said it once. Initially, it kept reloading the program and my screen kept flashing "beef" and I was really confused.
